websites/Z/ZeroTwo/dist/metadata.json

```
@@ -7,10 +7,11 @@
 	"service": "ZeroTwo",
 	"description": {
 		"en": "I'm a waifu Discord bot based on ZeroTwo (Darling in the FranXX). I serve a variety of needs and have tons of functionality you can try!",
-		"de": "Ich bin ein Waifu Discord Bot basierend auf ZeroTwo (Darling in the FranXX). Ich erfülle eine Vielzahl von Bedürfnissen und habe jede Menge Funktionen, die Sie ausprobieren können!"
+		"de": "Ich bin ein Waifu Discord Bot basierend auf ZeroTwo (Darling in the FranXX). Ich erfülle eine Vielzahl von Bedürfnissen und habe jede Menge Funktionen, die Sie ausprobieren können!",
+		"ja_JP": "私はZeroTwo（Darling in the FranXX）をベースにしたwaifu Discordボットです。 私はさまざまなニーズに対応し、試すことができる機能がたくさんあります。"
```

Upon talking with a Recodive proofreader, they believe this translation is poor and looks to be a Google translate. 
```suggestion
		"ja_JP": "「ダーリン・イン・ザ・フランキス」に登場するゼロツーをモチーフにした、Waifu Discordボットです。様々なシーンで使用できる機能や、たくさんの試せる機能があります。"
```

Unless reasoned otherwise, please use this translation.

---

yes, the translation is pretty bad. i didn't have time to write a good one because a macbook's keyboard isn't made to write kanji. i also didn't want to make another pr once i managed to write one myself so yeah, sorry